OE0-55: replace URL for IMIS CodeSystem

sniedzielski opened the pull request:
TICKET: https://openimis.atlassian.net/browse/OE0-55

dragos-dobre commented on api_fhir_r4/defaultConfig.py:
```diff
@@ -5,6 +5,7 @@
         "female": "F",
         "other": "O"
     },
+    "host_domain": "http://127.0.0.1:8000",
```
Try to get the host domain from "self.request.build_absolute_uri()". Thanks.

sniedzielski replied:
ok

sniedzielski commented:
@dragos-dobre @delcroip I used the approach proposed by @delcroip from that PR https://github.com/openimis/openimis-be_py/pull/53. On my computer tests passed when the change from main assembly module is applied. I also added full urls in references and I fixed bugs detected in some references.

sniedzielski commented:
@delcroip thx for merging 'SITE_URL' in assembly module. @dragos-dobre tests now passed in that PR. Please review and later I will check on release server how this site_url works. Thx in advance

sniedzielski commented on api_fhir_r4/tests/test/test_group.json:
```diff
@@ -14,7 +14,7 @@
                   "url":"https://openimis.github.io/openimis_fhir_r4_ig/StructureDefinition/address-location-reference",
                   "valueReference":{
-                     "reference":"Location/TEST_NAME-village"
+                     "reference":"/api/api_fhir_r4/Location/637f05cf-d8e8-4135-8250-7f01f01936bc"
```
ok

sniedzielski replied:
I will check this. I think I simply forgot remove this from fixtures/test payloads like here. But anyway I verify this if somewhere else we have such mistakes in references.

sniedzielski commented:
@dragos-dobre I fixed what you requested. please review again, thx!
